client/src/components/_common/TextCopyField/TextCopyField.js

```diff
@@ -39,7 +39,7 @@ const TextCopyField = ({ value, placeholder }) => {
         className="form-control"
         placeholder={placeholder}
         data-testid="input"
-        readOnly={isEmpty}
+        readOnly
```

Comment:
Made this `readOnly` to disable typing in this field, and the squigglies that the browser renders if it doesn't recognize a word. Copying directly is still possible